blocks/core/ff_module/ff_module-response-recipient-list/ff_module-response-recipient-list.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
 
 var React = require('react');
-
-var ProfileResponseButton = require('./../ff_module-profile-response-button/ff_module-profile-response-button'),
-    ItemRepeater = require('../../ff_container/ff_container-item-repeater/ff_container-item-repeater');
+var ProfileResponseButton = require('../ff_module-profile-response-button/ff_module-profile-response-button.js');
+var ItemRepeater = require('../../ff_container/ff_container-item-repeater/ff_container-item-repeater.js');
 
 module.exports = React.createClass({
     displayName: 'TaskResponseRecipientList',
     propTypes:{
         responses: React.PropTypes.array.isRequired,
-        onSelect: React.PropTypes.func.isRequired
+        onSelect: React.PropTypes.func.isRequired,
+        currentTaskResponse: React.PropTypes.element
     },
     render: function(){
-        return (
-            <ItemRepeater modifier="separated">
-                {this.props.responses && this.props.responses.map(response =>
-                    <ProfileResponseButton
+        return <ItemRepeater modifier="separated">
+                {this.props.responses && this.props.responses.map(response =>{
+                	var currentResponse = response.isSelected ? <div className='ff_module-response-recipient-list__currentResponse'>{this.props.currentTaskResponse}</div> : null;
+                    return <div><ProfileResponseButton
```

Comment:
On line 18, the `div` will need a key as it's inside a map...you could take the key from line 19 and use it here instead, as it's the outer item now.